cost-analyzer/templates/grafana-dashboard-cluster-metrics-template.yaml

```diff
@@ -8,7 +8,9 @@ metadata:
   name: cluster-metrics-dashboard
   labels:
     {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
-    grafana_dashboard: "1"
+    {{- if $.Values.grafana.sidecar.dashboards.label }}
```

Comment:
This LGTM, but do you mind adding an 'else' clause that uses grafana_dashboard if the values doesn't exist, for backwards compatibility with old values.yaml?